addons/website_slides/models/slide_question.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, fields, models, SUPERUSER_ID, _
+from odoo.exceptions import ValidationError
+
+
+
+class SlideQuestion(models.Model):
+    _name = "slide.question"
+    _rec_name = "question"
+    _description = """
+        Question for a slide. A slide can have multiple questions and each question
+        must have at least 2 possible answers and only one good answer.
+        """
+
+    sequence = fields.Integer("Sequence", default=10)
+    question = fields.Char(required=True, string="Question Name")
+    slide_id = fields.Many2one('slide.slide', string="Slide")
+    answer_ids = fields.One2many('slide.answer', 'question_id', string="Answer")
+
+    @api.constrains('answer_ids')
+    def _check_only_one_good_answer(self):
+        for question in self:
+            good_answer_count = 0
+            for answer in question.answer_ids:
+                if answer.is_correct:
+                    good_answer_count += 1
+                    if good_answer_count > 1:
+                        raise ValidationError(_('A question can only have one good answer'))
+
+    @api.constrains('answer_ids')
+    def _check_correct_answer(self):
+        for question in self:
+            if not any([answer.is_correct for answer in question.answer_ids]):
```

Comment:
imo you could check this condition within "_check_only_one_good_answer" and rename it "_check_good_answers_count"